home-manager/linux-ci.nix

```diff
@@ -13,14 +13,27 @@
 
 {
   home = {
-    packages = with pkgs.unstable; [
-      # zed-editor # Disabling because of it is most flaky in unstable channels
+    packages =
+      (with pkgs.unstable; [
+        # Disabling zed-editor because of it is most flaky in unstable channels.
+        # If you want to use newer versions, consider to use upstream providing binary cache
+        # See https://github.com/zed-industries/zed/issues/19937 for detail
+        ## zed-editor
 
-      typos-lsp
-      dprint
-      # gnome-boxes # Omitting. I believe it is a core package of NixOS.
-      shogihome
-      apery
-    ];
+        typos-lsp
+        dprint
+        # gnome-boxes # Omitting. I believe it is a core package of NixOS.
+      ])
+      # Test builds and push the binary cache from CI
+      ++ (with pkgs.patched; [
+        lima
+        signal-desktop
+        shogihome
+      ])
+      ++ [
+        # Also required the binary cache by special reasons
+        # TODO: Remove after nixos-25.11. See overlays for detail
+        pkgs.gnome-keyring
```

Comment:
Hmm... the build in local takes longtime, however that overlay seems not working in GHA CI 🤔 
(Copying from official binary cache)

```
copying path '/nix/store/8lw10qpjyq4y8f03c5588iqgcf1wscvw-gnome-keyring-48.0' from 'https://cache.nixos.org/'...
```